### PR TITLE
check SDK version before releasing a minor

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- check SDK version before releasing a minor 
- fixed some minor style issues

**Status (please check what you already did)**:
- ~[ ] added some tests for the functionality~
- ~[ ] updated the documentation~
- ~[ ] updated the changelog (please check [changelog]~(https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by federicotdn** [comment]
@tmbo Should I review this PR?

File: scripts/release.py [hunk 1]
```diff
@@ -120,6 +120,39 @@ def is_valid_version_number(v: Text) -> bool:
         sys.exit(1)
 
 
+def get_rasa_sdk_version() -> Text:
+    """Find out what the referenced version of the Rasa SDK is."""
+
+    env_file = project_root() / "requirements.txt"
+
+    with env_file.open() as f:
+        for line in f.readlines():
```
Comment (federicotdn) [inline scripts/release.py]: Optional:
```suggestion
        for line in f:
```

File: scripts/release.py [hunk 2]
```diff
@@ -120,6 +120,39 @@ def is_valid_version_number(v: Text) -> bool:
...
+            if "rasa-sdk" in line:
+                version = line.split("=")[-1]
+                return version.strip()
+        else:
+            raise Exception("Failed to find Rasa SDK version in requirements.txt")
+
+
+def validate_code_is_release_ready(version: Text) -> None:
+    """Make sure the code base is valid (e.g. rasa sdk is up to date)."""
```
Comment (federicotdn) [inline scripts/release.py]:
```suggestion
    """Make sure the code base is valid (e.g. Rasa SDK is up to date)."""
```

**Comment by tmbo** [comment]
ah now I get why you asked about the review, sorry used the wrong dropdown to select you and assigned you instead of requesting a review 🙈

**Pull request merged**